vendor/engines/secure_rooms/app/views/secure_rooms/indala_numbers/edit.html.haml

```diff
@@ -0,0 +1,17 @@
+= content_for :head_content do
+  = render "users/form_head"
+
+= content_for :h1 do
+  = current_facility
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "users"
+
+%h2= text("views.users.edit.head", user: @user.full_name)
+%p= text("views.users.edit.subhead")
```

Comment:
I think we can kill this text. In this context "You may use this to edit external users. The email address and username must match." isn't relevant.